sdk/opa_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package sdk
```

Comment:
This looks good for unit testing the SDK struct. I think we should add another example test to show people how to use the SDK struct. See https://github.com/open-policy-agent/opa/blob/master/rego/example_test.go for examples of example tests (😅 )

Reply:
Good idea! Will add on Monday.